pkg/operator/v410_00_assets/bindata.go

```diff
@@ -468,6 +468,30 @@ spec:
         name: resource-dir
       - mountPath: /etc/kubernetes/static-pod-certs
         name: cert-dir
+  - name: kube-scheduler-recovery-controller
+    env:
+    - name: POD_NAMESPACE
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.namespace
+    image: ${OPERATOR_IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command: ["/bin/bash", "-euxo", "pipefail", "-c"]
+    args:
+      - |
+        timeout 3m /bin/bash -exuo pipefail -c 'while [ -n "$(ss -Htanop \( sport = 10443 \))" ]; do sleep 1; done'
```

Comment:
/hold

this was changed to 11443

Reply:
replaced with https://github.com/openshift/cluster-kube-scheduler-operator/pull/315
/close